django/utils/timezone.py

```diff
@@ -295,13 +295,15 @@ def template_localtime(value, use_tz=None):
 
 # Utilities
 
-def localtime(value, timezone=None):
+def localtime(value=None, timezone=None):
     """
     Converts an aware datetime.datetime to local time.
 
     Local time is defined by the current time zone, unless another time zone
     is specified.
     """
```

Comment:
This docstring could use another sentence explaining the new `now` functionality. Something like "If the `value` parameter is omitted, returns the current time in the current timezone."